target/linux/rockchip/armv8/base-files/etc/hotplug.d/net/40-net-smp-affinity

```diff
@@ -43,6 +43,7 @@ friendlyarm,nanopi-r4s-enterprise)
 	set_interface_core 10 "eth0"
 	set_interface_core 20 "eth1"
 	;;
+friendlyarm,nanopc-t6|\
```

Comment:
ditto

Reply:
r2c usb eth1

Reply:
the whole section should be put above `friendlyarm,nanopi-r2c`